README.md

```diff
@@ -11,19 +11,19 @@ more flexible than standard-template charts provided by charting libraries. You
 
 Philosophy
 ---
-Plottable's core philosophy is "Composition over Configuration", so a lot of the API flexibility is in choosing which `Components` to use, and how to arrange them in `Tables`, rather than setting high-level properties on the charts. If you find you need a feature that doesn't exist, consider writing a new `Component` that implements it as a plugin. This way, you can get your custom functionality and still benefit from the rest of the library. Drop a line at our [Google group](https://groups.google.com/forum/#!forum/plottablejs) if you want help implementing a plugin.
+Plottable's core philosophy is "Composition over Configuration", so a lot of the API flexibility is in choosing which `Components` to use, and how to arrange them in `Tables`, rather than setting high-level properties on the charts. If you find you need a feature that doesn't exist, consider writing a new `Component` that implements. This way, you can get your custom functionality and still benefit from the rest of the library.
```

Comment:
consider writing a new `Component` that implements.... the functionality that is missing.

The sentence ends abruptly, and needs a tiny bit more.